dev-util/ragel/ragel-6.9.ebuild

```diff
@@ -14,7 +14,7 @@ LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~m68k-mint ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
 IUSE="vim-syntax"
-
+CXXFLAGS="${CXXFLAGS} -std=c++98"
```

Comment:
You are not allowed to modify `$CXXFLAGS` in global scope.

Here's the idea:
1) `inherit flag-o-matic`
2) add in `src_configure`, before actually calling the configure program/script:
```
# Switch back to C++98 explicitly overflow detected
# due to -Wnarrowing in -std=c++11/14
# See also bug #582606
append-cxxflags -std=c++98
```
Once you've done that, squash the commits into one. You can reformat commit messages and merge commits for the same ebuild (i.e. into one logical change) using `git rebase -i` (http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html).